### ISSUE TITLE
Feature: execute run-document-server.sh as nonroot

### ISSUE THREAD

**Issue by MnrGreg** [ONLYOFFICE/Docker-DocumentServer#172]
Feature request:

The 'ENTRYPOINT run-document-server.sh' currently runs as root. Good container platform practices prevent the execution of container running as root. Most enterprise Kubernetes distributions enforce this through Pod Security Policies.

Adding 'USER ds' prior to the 'ENTRYPOINT' forces the script to run as the ds user, however, many subsequent steps fail.

The ask is for this container permission structure be updated to run as nonroot.

**Comment by ShockwaveNN** [comment]
@MnrGreg Thanks for your proposal, we'll think about it

**Comment by durandx** [comment]
Hello, 

Do you have any update about this enhancement ?
Thanks.

**Comment by SuperSandro2000** [comment]
Then the default port probably couldn't  be 80. See https://github.com/ONLYOFFICE/Docker-DocumentServer/blob/master/run-document-server.sh#L16

**Comment by edvinkuric** [comment]
> Then the default port probably couldn't be 80. Source https://github.com/ONLYOFFICE/Docker-DocumentServer/blob/master/run-document-server.sh#L16

Yes that's true - I would also love to run this setup as non-root 

Thank you very much :)

**Comment by igwyd** [comment]
Ticket #66316

**Comment by beckjin** [comment]
Hello, 

Is there a plan for this requirement?

**Comment by igwyd** [comment]
@beckjin unfortunately there is no news yet. But we have another project with separated services by containers, they are all runs as nonroot https://github.com/ONLYOFFICE/Docker-Docs. This might be something that interests you.
